presentation/common/src/presentation-common/rules/Rule.ts

```diff
@@ -19,7 +19,7 @@ import { RequiredSchemaSpecification } from "./SchemasSpecification";
  */
 export interface RuleBase {
   /** Used for serializing to JSON. */
-  ruleType: RuleTypes;
+  ruleType: `${RuleTypes}`;
```

Comment:
In places where you've replaced an `enum` type with a template literal, please include a doc link to the `enum` so people can easily find it when browsing the documentation site, as I assume (@ben-polinsky confirm?) that the docs build will not generate one.
In future consider preferring string union over string enum.

Reply:
> In places where you've replaced an enum type with a template literal, please include a doc link to the enum so people can easily find it when browsing the documentation site

Added. 

> In future consider preferring string union over string enum.

Generally that's what we do. These were leftovers from the initial implementation, so I had to keep the enums to avoid breaking current uses. On the other hand, I'd argue that using them like this instead of a string union makes it easier to document each value, especially if the type is used in more than one place.

Reply:
Did you mean `@see [[RuleTypes]]`?

Reply:
Looks like docs builder  understands the link without the brackets too:
- [source](https://github.com/iTwin/itwinjs-core/blob/8737e4cc516e0f410f829d5918b246d5bad3d655/core/frontend/src/ImageUtil.ts#L244)
- [result](https://www.itwinjs.org/reference/core-frontend/rendering/tryimageelementfromurl/)

Reply:
Yes but what is the scope? What does it do if you provide more than one word after `@see`, or the name of a symbol in a different file, or in a different package? @ben-polinsky might know - I certainly don't.
`[[Symbol]]` and `[Symbol]($package)` are unambiguous.

Reply:
I would guess it's the same as `[[Symbol]]`. I don't mind changing it to that format, but I managed to get 2 of the 4 flaky CI builds to pass on this PR, so would like to avoid having to restart again... I can replace the `@see Symbol` with `@see [[Symbol]]` across the whole repository in a separate PR afterwards.